src/libraries/Microsoft.PowerFx.Core/Functions/BuiltinFunction.cs

```diff
@@ -11,13 +11,21 @@ namespace Microsoft.PowerFx.Core.Functions
     // Abstract base class for all Texl builtin functions.
     internal abstract class BuiltinFunction : TexlFunction
     {
-        public static readonly DName OneColumnTableResultName = new DName("Result");
-        public static readonly DName ColumnName_Name = new DName("Name");
-        public static readonly DName ColumnName_Address = new DName("Address");
-        public static readonly DName ColumnName_Value = new DName("Value");
-        public static readonly DName ColumnName_FullMatch = new DName("FullMatch");
-        public static readonly DName ColumnName_SubMatches = new DName("SubMatches");
-        public static readonly DName ColumnName_StartMatch = new DName("StartMatch");
+        public const string OneColumnTableResultNameStr = "Result";
+        public const string ColumnName_NameStr = "Name";
```

Comment:
should these literals be private? are they used elsewhere, and what was the benefit of splitting them like this?

Reply:
The literals like Result and Value are used sometimes in the interpreter when reading or creating one-column tables.

Reply:
ah, I see.